### PR TITLE
html-movie-seat-booking task

### PR DESCRIPTION
**Pull request by BlueLamantine**
Please, review my task!
[DEMO](https://bluelamantine.github.io/cinema-booking/)
[CODE](https://github.com/BlueLamantine/cinema-booking)

### CODE REVIEW

File: submissions/BlueLamantine/index.html [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+    </header>
+
+    <main>
+      <form id="ticketsForm" class="booking">
+        <section class="calendar">
+          <fieldset class="booking__date">
+            <legend class="booking__headline">Date</legend>
+
+            <div class="slider__date">
+              <div
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/index.html]: Why you don't use `button` for this element?

File: submissions/BlueLamantine/index.html [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+                class="slider__button slider__button--prev"
+                aria-label="Previous"
+                tabindex="0"
+              >
+                <i class="fa fa-chevron-left"></i>
+              </div>
+              <label for="tue" class="radio__lable">
+                <input type="radio" name="day" id="tue" class="main__radio" />
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/index.html:89]: 🔥

File: submissions/BlueLamantine/index.html [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+        </section>
+      </form>
+    </main>
+    <footer>
+      <p class="footer__info">
+        Made by
+        <a href="https://github.com/BlueLamantine" class="ghlink">
+          BlueLamantine
+        </a>
+      </p>
```
Comment (A-Ostrovnyy) [inline submissions/BlueLamantine/index.html:674]: This alignment a bug or a feature?

image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good  work!

File: submissions/BlueLamantine/index.html (hunk 1 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/index.html]: A lot of slider-libraries use div for creating slider elements. But OK, now it is a ```button``` with type=button (cause I want to use other buttons in a form without it submitting)

File: submissions/BlueLamantine/index.html (hunk 2 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/index.html:89]: =)

File: submissions/BlueLamantine/index.html (hunk 3 above)
Comment (BlueLamantine) [reply submissions/BlueLamantine/index.html:674]: this is a part of my design

**Comment by BlueLamantine** [comment]
@A-Ostrovnyy thanks for attention to my homework! I followed your recommendation

File: submissions/BlueLamantine/index.html (hunk 3 above)
Comment (A-Ostrovnyy) [reply submissions/BlueLamantine/index.html:674]: Ok)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**